### ISSUE TITLE
ImportError: DLL load failed: The specified module could not be found

### ISSUE THREAD

**Issue by Krprashant94** [pytorch/pytorch#9071]
## Python Import Error with a message " ImportError: DLL load failed: The specified module could not be found. "

I am installed pytorch using
pip3 install http://download.pytorch.org/whl/cpu/torch-0.4.0-cp36-cp36m-win_amd64.whl 
on plan python (not conda), and running hello world program and getting the same error.

## Code 
```
from __future__ import print_function
import torch
```
(as given at https://pytorch.org/tutorials/beginner/blitz/tensor_tutorial.html#sphx-glr-beginner-blitz-tensor-tutorial-py)

## Error
 ```

Traceback (most recent call last):
  File "C:\Users\Prashant\Desktop\test.py", line 9, in <module>
    import torch
  File "C:\Python\lib\site-packages\torch\__init__.py", line 78, in <module>
    from torch._C import *
ImportError: DLL load failed: The specified module could not be found.
```

## System Info

> OS: Microsoft Windows 10 Pro
> GCC version: (tdm64-1) 5.1.0
> CMake version: version 3.12.0-rc2
> 
> Python version: 3.6
> Is CUDA available: No, I am using AMD
> CUDA runtime version: No CUDA
> GPU models and configuration: No CUDA
> Nvidia driver version: No CUDA
> cuDNN version: No CUDA
> 
> Versions of relevant libraries:
> [pip3] numpy (1.14.5)
> [pip3] numpydoc (0.8.0)
> [pip3] torch (0.4.0)
> [pip3] torchvision (0.2.1)
> [conda] Could not collect

## I already tried below answers 

https://github.com/pytorch/pytorch/issues/4518
https://github.com/peterjc123/pytorch-scripts/issues/26
https://discuss.pytorch.org/t/windows-10-cant-import-torch/18100 (installed both)

rename the _C not working in my case

*Most of the answer is for conda ,linux and NVIDA so some of them are not relevant 

Screenshot of torch folder
![2018-07-01](https://user-images.githubusercontent.com/12370519/42128896-631327dc-7cd3-11e8-88de-065ea01225fb.png)

**Comment by SsnL** [comment]
cc @peterjc123

**Comment by peterjc123** [comment]
@Krprashant94 Have you tried this https://gist.github.com/peterjc123/6b804651288e76db7b5fabe5348e1f03?

**Comment by Krprashant94** [comment]
No,not got success,, I used 
pip install mkl mkl-fft intel-openmp numpy
pip install psutil first
pip install mkl
pip install mkl-fft

# Code
`import psutil, os
p = psutil.Process( os.getpid() )
for dll in p.memory_maps():
    print(dll.path)`

# Output
`
C:\Python\python.exe
C:\Python\DLLs\_bz2.pyd
C:\Python\DLLs\select.pyd
C:\Python\DLLs\_socket.pyd
C:\Python\python36.dll
C:\Windows\System32\locale.nls
C:\Windows\Globalization\Sorting\SortDefault.nls
C:\Windows\System32\en-US\KernelBase.dll.mui
C:\Windows\System32\en-US\kernel32.dll.mui
C:\Python\DLLs\_lzma.pyd
C:\Windows\System32\version.dll
C:\Python\Lib\site-packages\psutil\_psutil_windows.cp36-win_amd64.pyd
C:\Python\python3.dll
C:\Python\vcruntime140.dll
C:\Windows\System32\wtsapi32.dll
C:\Windows\System32\rsaenh.dll
C:\Windows\System32\IPHLPAPI.DLL
C:\Windows\System32\cryptbase.dll
C:\Windows\System32\cryptsp.dll
C:\Windows\System32\bcrypt.dll
C:\Windows\System32\fltLib.dll
C:\Windows\System32\powrprof.dll
C:\Windows\System32\profapi.dll
C:\Windows\System32\kernel.appcore.dll
C:\Windows\System32\cfgmgr32.dll
C:\Windows\System32\KernelBase.dll
C:\Windows\System32\win32u.dll
C:\Windows\System32\msvcp_win.dll
C:\Windows\System32\windows.storage.dll
C:\Windows\System32\gdi32full.dll
C:\Windows\System32\ucrtbase.dll
C:\Windows\System32\bcryptprimitives.dll
C:\Windows\System32\kernel32.dll
C:\Windows\System32\gdi32.dll
C:\Windows\System32\psapi.dll
C:\Windows\System32\ws2_32.dll
C:\Windows\System32\shlwapi.dll
C:\Windows\System32\rpcrt4.dll
C:\Windows\System32\sechost.dll
C:\Windows\System32\combase.dll
C:\Windows\System32\SHCore.dll
C:\Windows\System32\user32.dll
C:\Windows\System32\shell32.dll
C:\Windows\System32\advapi32.dll
C:\Windows\System32\msvcrt.dll
C:\Windows\System32\imm32.dll
C:\Windows\System32\ntdll.dll`

**Comment by Krprashant94** [comment]
Okey one ore thing : It may be relevent 
When i am copying these file to /python36/DLLs/ the error messaged changed to 
> libiomp5md.dll
> libiomp5md.pdb
> libiompstubs5md.dll


```
Traceback (most recent call last):
  File "C:\Users\Prashant\Desktop\test.py", line 3, in <module>
    import torch
  File "C:\Python\lib\site-packages\torch\__init__.py", line 78, in <module>
    from torch._C import *
ImportError: numpy.core.multiarray failed to import
```
I did as https://github.com/pytorch/pytorch/issues/2731 and now 

```
  File "C:\Users\Prashant\Desktop\test.py", line 3, in <module>
    import torch
  File "C:\Python\lib\site-packages\torch\__init__.py", line 78, in <module>
    from torch._C import *
ImportError: DLL load failed: The specified module could not be found.
```

**Comment by peterjc123** [comment]
For your recent post:
Why? It should't happen if you've set the environmental variable `PATH` correctly. Did you skipped that part in my first link?
For your previous post:
Can you try `import numpy` first? And then my question is that have you installed VC2017 Redist? And finally, you can try lucasg/Dependencies on Github, use it to open _C.pyd and give me the log.

**Comment by Krprashant94** [comment]
Okay, thanks a lot I was written the wrong path and tried almost all of the answer in online.
Thankyou

**Comment by hemangjoshi37a** [comment]
## Code : 
```
import psutil, os
p = psutil.Process( os.getpid() )
for dll in p.memory_maps():
  print(dll.path)

```
## Output
``
runfile('C:/Users/hj/autokeras_test.py', wdir='C:/Users/hj')
C:\Windows\System32\locale.nls
C:\Windows\Globalization\Sorting\SortDefault.nls
C:\Windows\System32\en-US\KernelBase.dll.mui
C:\Windows\System32\en-US\kernel32.dll.mui
C:\Users\hj\Anaconda3\pythonw.exe
C:\Users\hj\Anaconda3\python36.dll
C:\Users\hj\Anaconda3\DLLs\_sqlite3.pyd
C:\Users\hj\Anaconda3\DLLs\_decimal.pyd
C:\Users\hj\Anaconda3\pkgs\pywin32-223-py36hfa6e2cd_0\Library\bin\pythoncom36.dll
C:\Users\hj\Anaconda3\DLLs\_multiprocessing.pyd
C:\Users\hj\Anaconda3\DLLs\_overlapped.pyd
C:\Users\hj\Anaconda3\DLLs\_elementtree.pyd
C:\Users\hj\Anaconda3\pkgs\pywin32-223-py36hfa6e2cd_0\Lib\site-packages\win32\win32api.pyd
C:\Users\hj\Anaconda3\pkgs\python-3.6.6-he025d50_0\DLLs\unicodedata.pyd
C:\Users\hj\Anaconda3\DLLs\_bz2.pyd
C:\Users\hj\Anaconda3\DLLs\_ctypes.pyd
C:\Users\hj\Anaconda3\DLLs\select.pyd
C:\Users\hj\Anaconda3\Library\bin\pywintypes36.dll
C:\Users\hj\Anaconda3\DLLs\_socket.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\win32\_win32sysloader.pyd
C:\Windows\System32\kernel32.dll
C:\Windows\System32\user32.dll
C:\Windows\System32\ntdll.dll
C:\Windows\System32\psapi.dll
C:\Users\hj\Anaconda3\Library\bin\libiomp5md.dll
C:\Users\hj\Anaconda3\Library\bin\mklml.dll
C:\Users\hj\Anaconda3\pkgs\.trash\c77b438f-8b95-4c6b-b3f4-c3f5e071ac94
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5l.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5o.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5i.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5g.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5f.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5ds.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5d.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5p.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5t.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Library\bin\hdf5.dll
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\ndimage\_ni_label.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\ndimage\_nd_image.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\DLLs\sqlite3.dll
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5ac.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5a.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\pkgs\.trash\4f7550e6-c388-48ce-a996-b5c3afee535a
C:\Users\hj\Anaconda3\Library\bin\libexslt.dll
C:\Users\hj\Anaconda3\pkgs\.trash\faa58ceb-3deb-4216-b6cc-b71765189641
C:\Users\hj\Anaconda3\pkgs\.trash\e65fd056-165d-4c7f-9362-48a0e695b680
C:\Users\hj\Anaconda3\pkgs\.trash\760f1e71-d492-4486-916a-d253ed631cd8
C:\Users\hj\Anaconda3\Library\bin\libxslt.dll
C:\Users\hj\Anaconda3\pkgs\.trash\d3d5372e-94f1-4ee6-8cbd-56247851f44a
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5s.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Library\bin\hdf5_hl.dll
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\defs.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\_objects.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5r.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\pkgs\.trash\3b4940da-01cf-4b18-bfda-5bc04c2fe2ca
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\spatial\_hausdorff.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\spatial\_distance_wrap.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\spatial\_voronoi.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\spatial\qhull.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\spatial\ckdtree.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\interpolate\interpnd.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\interpolate\_ppoly.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\interpolate\_bspl.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\interpolate\dfitpack.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\interpolate\_fitpack.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\optimize\_nnls.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\optimize\_lsq\givens_elimination.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\optimize\_group_columns.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\optimize\_minpack.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\sparse\linalg\eigen\arpack\_arpack.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\sparse\csgraph\_reordering.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\sparse\csgraph\_min_spanning_tree.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\sparse\csgraph\_traversal.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\sparse\csgraph\_tools.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\sparse\csgraph\_shortest_path.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\sparse\_csparsetools.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\sparse\_sparsetools.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\integrate\lsoda.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\integrate\_dop.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\integrate\vode.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\integrate\_quadpack.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\integrate\_odepack.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\special\_ellip_harm_2.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\special\_comb.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\special\specfun.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\special\_ufuncs_cxx.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Library\bin\libmmd.dll
C:\Users\hj\Anaconda3\Library\bin\libifcoremd.dll
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\special\_ufuncs.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\linalg\cython_lapack.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\linalg\cython_blas.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\linalg\_decomp_update.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\linalg\_solve_toeplitz.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\linalg\_flinalg.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\linalg\_flapack.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Library\bin\mkl_rt.dll
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\linalg\_fblas.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\_lib\_ccallback_c.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\testing.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\util\_move.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\io\msgpack\_unpacker.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\io\msgpack\_packer.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\writers.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\json.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\parsers.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\optimize\_slsqp.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\reshape.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\groupby.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\reduction.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\skiplist.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\window.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\optimize\_cobyla.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\pkgs\.trash\5ad030ba-a84d-4807-a3bf-b2133b906c80
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\optimize\_zeros.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\optimize\moduleTNC.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\optimize\_lbfgsb.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\optimize\minpack2.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\sparse\linalg\dsolve\_superlu.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\scipy\sparse\linalg\isolve\_iterative.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\sparse.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\internals.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\indexing.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\interval.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\ops.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\join.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\offsets.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\resolution.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\frequencies.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\period.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\index.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\bottleneck\move.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\bottleneck\nonreduce_axis.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\bottleneck\reduce.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\hashing.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\properties.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\algos.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\lib.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\missing.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\hashtable.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\fields.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\timestamps.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\strptime.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\ccalendar.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\parsing.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\timezones.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\timedeltas.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\nattype.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\conversion.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\pkgs\.trash\faabaffb-3915-4520-ad39-5bff99d60079
C:\Users\hj\Anaconda3\pkgs\.trash\9787515b-b6cc-49be-bb68-cff68f96ac0a
C:\Users\hj\Anaconda3\pkgs\.trash\cbb0530a-8a3f-4e7e-adaa-10503941dd63
C:\Users\hj\Anaconda3\pkgs\.trash\416772ec-231f-4b55-96e8-aed3fc7959b9
C:\Users\hj\Anaconda3\Lib\site-packages\zmq\backend\cython\_device.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\zmq\backend\cython\_version.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\zmq\backend\cython\_poll.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\zmq\backend\cython\utils.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\zmq\backend\cython\socket.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\zmq\backend\cython\context.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\zmq\backend\cython\message.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Library\bin\libzmq-mt-4_2_5.dll
C:\Users\hj\Anaconda3\Lib\site-packages\zmq\backend\cython\error.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\zmq\backend\cython\constants.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\bottleneck\nonreduce.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslib.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\pkgs\.trash\8dc71247-cacf-41be-80d5-8c948f62836e
C:\Users\hj\Anaconda3\pkgs\.trash\7167f57d-f034-4cd6-8ff6-e3d40fe4a0c6
C:\Users\hj\Anaconda3\pkgs\.trash\4210679c-73fe-41ba-89c4-bb3302cf98a8
C:\Users\hj\Anaconda3\pkgs\.trash\34cbedf6-069b-4092-96a5-9cc073d8a652
C:\Users\hj\Anaconda3\pkgs\openssl-1.0.2p-hfa6e2cd_0\Library\bin\libeay32.dll
C:\Users\hj\Anaconda3\DLLs\_ssl.pyd
C:\Users\hj\Anaconda3\DLLs\_hashlib.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\torch\_nvrtc.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\pkgs\.trash\2cbcfc16-5452-444d-b6a6-bd718dd85e8f
C:\Users\hj\Anaconda3\pkgs\.trash\4a5c9caa-bf3e-422d-b612-9b75ed4a72aa
C:\Users\hj\Anaconda3\Lib\site-packages\kiwisolver.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\pkgs\.trash\964b13f8-19b1-49d5-a57d-60684a9d7449
C:\Users\hj\Anaconda3\pkgs\.trash\ad0b6574-5754-4273-9b06-05c6f444c909
C:\Users\hj\Anaconda3\pkgs\.trash\3d047e0c-3bba-4140-be24-94b2966355bc
C:\Users\hj\Anaconda3\pkgs\.trash\b2e302e1-7afc-457e-81e4-706b85e392ef
C:\Users\hj\Anaconda3\pkgs\.trash\75888ce3-b976-4947-8314-ad747e0fdf88
C:\Users\hj\Anaconda3\Lib\site-packages\pandas\_libs\tslibs\np_datetime.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\DLLs\pyexpat.pyd
C:\Users\hj\Anaconda3\pkgs\openssl-1.0.2p-hfa6e2cd_0\Library\bin\ssleay32.dll
C:\Users\hj\Anaconda3\pkgs\cryptography-2.3.1-py36h74b6da3_0\Lib\site-packages\cryptography\hazmat\bindings\_openssl.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\ucrtbase.dll
C:\Users\hj\Anaconda3\pkgs\cffi-1.11.5-py36hfa6e2cd_1\Lib\site-packages\_cffi_backend.cp36-win_amd64.pyd
C:\Windows\System32\msvcp140.dll
C:\Users\hj\Anaconda3\DLLs\_asyncio.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\_yaml.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Library\bin\yaml.dll
C:\Users\hj\Anaconda3\DLLs\_lzma.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\tornado\speedups.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Library\bin\libpng16.dll
C:\Users\hj\Anaconda3\Library\bin\zlib.dll
C:\Windows\System32\api-ms-win-crt-utility-l1-1-0.dll
C:\Users\hj\Anaconda3\pkgs\cryptography-2.3.1-py36h74b6da3_0\Lib\site-packages\cryptography\hazmat\bindings\_constant_time.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\_conv.cp36-win_amd64.pyd
C:\Program Files\AVAST Software\Avast\x64\aswhooka.dll
C:\Users\hj\Anaconda3\api-ms-win-crt-filesystem-l1-1-0.dll
C:\Users\hj\Anaconda3\python3.dll
C:\Users\hj\Anaconda3\api-ms-win-crt-process-l1-1-0.dll
C:\Users\hj\Anaconda3\api-ms-win-crt-conio-l1-1-0.dll
C:\Users\hj\Anaconda3\api-ms-win-crt-time-l1-1-0.dll
C:\Users\hj\Anaconda3\api-ms-win-crt-environment-l1-1-0.dll
C:\Users\hj\Anaconda3\api-ms-win-crt-math-l1-1-0.dll
C:\Users\hj\Anaconda3\api-ms-win-crt-locale-l1-1-0.dll
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5fd.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\_errors.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\api-ms-win-crt-stdio-l1-1-0.dll
C:\Users\hj\Anaconda3\api-ms-win-crt-convert-l1-1-0.dll
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\_proxy.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\api-ms-win-crt-string-l1-1-0.dll
C:\Users\hj\Anaconda3\api-ms-win-crt-heap-l1-1-0.dll
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\h5z.cp36-win_amd64.pyd
C:\Windows\System32\wsock32.dll
C:\Users\hj\Anaconda3\api-ms-win-core-processthreads-l1-1-1.dll
C:\Users\hj\Anaconda3\api-ms-win-core-file-l1-2-0.dll
C:\Users\hj\Anaconda3\api-ms-win-core-localization-l1-2-0.dll
C:\Users\hj\Anaconda3\api-ms-win-core-synch-l1-2-0.dll
C:\Users\hj\Anaconda3\vcruntime140.dll
C:\Users\hj\Anaconda3\api-ms-win-core-file-l2-1-0.dll
C:\Users\hj\Anaconda3\api-ms-win-crt-runtime-l1-1-0.dll
C:\Users\hj\Anaconda3\api-ms-win-core-timezone-l1-1-0.dll
C:\Users\hj\Anaconda3\Lib\site-packages\h5py\utils.cp36-win_amd64.pyd
C:\Users\hj\Anaconda3\Lib\site-packages\psutil\_psutil_windows.cp36-win_amd64.pyd
C:\Windows\System32\msvcr120.dll
C:\Windows\System32\dhcpcsvc6.dll
C:\Windows\System32\dhcpcsvc.dll
C:\Windows\System32\winnsi.dll
C:\Windows\System32\IPHLPAPI.DLL
C:\Windows\System32\uxtheme.dll
C:\Windows\System32\powrprof.dll
C:\Windows\System32\wtsapi32.dll
C:\Windows\System32\version.dll
C:\Windows\System32\WSHTCPIP.DLL
C:\Windows\System32\rsaenh.dll
C:\Windows\System32\mswsock.dll
C:\Windows\System32\cryptsp.dll
C:\Windows\System32\wship6.dll
C:\Windows\System32\secur32.dll
C:\Windows\System32\sspicli.dll
C:\Windows\System32\apphelp.dll
C:\Windows\System32\cryptbase.dll
C:\Windows\System32\msasn1.dll
C:\Windows\System32\cfgmgr32.dll
C:\Windows\System32\devobj.dll
C:\Windows\System32\crypt32.dll
C:\Windows\System32\KernelBase.dll
C:\Windows\System32\imm32.dll
C:\Windows\System32\ws2_32.dll
C:\Windows\System32\msctf.dll
C:\Windows\System32\ole32.dll
C:\Windows\System32\rpcrt4.dll
C:\Windows\System32\sechost.dll
C:\Windows\System32\oleaut32.dll
C:\Windows\System32\gdi32.dll
C:\Windows\System32\wininet.dll
C:\Windows\System32\setupapi.dll
C:\Windows\System32\lpk.dll
C:\Windows\System32\shell32.dll
C:\Windows\System32\iertutil.dll
C:\Windows\System32\advapi32.dll
C:\Windows\System32\usp10.dll
C:\Windows\System32\imagehlp.dll
C:\Windows\System32\msvcrt.dll
C:\Windows\System32\nsi.dll
C:\Windows\System32\urlmon.dll
C:\Windows\System32\shlwapi.dll
C:\Windows\System32\apisetschema.dll``

## Error : 
Related to : 
``
runfile('C:/Users/hj/autokeras_test.py', wdir='C:/Users/hj')
Traceback (most recent call last):

  File "<ipython-input-25-fd5ffb431523>", line 1, in <module>
    runfile('C:/Users/hj/autokeras_test.py', wdir='C:/Users/hj')

  File "C:\Users\hj\Anaconda3\lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 668, in runfile
    execfile(filename, namespace)

  File "C:\Users\hj\Anaconda3\lib\site-packages\spyder_kernels\customize\spydercustomize.py", line 108, in execfile
    exec(compile(f.read(), filename, 'exec'), namespace)

  File "C:/Users/hj/autokeras_test.py", line 1, in <module>
    import torch

  File "C:\Users\hj\Anaconda3\lib\site-packages\torch\__init__.py", line 78, in <module>
    from torch._C import *

ImportError: DLL load failed: The operating system cannot run %1.``

# Please help me

**Comment by peterjc123** [comment]
@hegemanjwh2 See #7579.

**Comment by hemangjoshi37a** [comment]
@peterjc123 ok sir...

**Comment by soumith** [comment]
@hegemanjwh2 probably typo. He meant @hemangjoshi37a :)

**Comment by sykrn** [comment]
@peterjc123  I found that this error, `from torch._C import * (ImportError: DLL load failed: The specified module could not be found`, rises when I installed Pytorch outside the base environment (using anaconda).   But, when I installed Pytorch in the base/main environment, all are okay.
I guess that because only the base environment is included in PATH variable. i.e. 
>[PythonRoot]
>[PythonRoot]\Library\bin
>[PythonRoot]\Scripts
>[PythonRoot]\bin

where `PythonRoot=Anaconda3`, (main anaconda directory). 

I don't know what's wrong with the anaconda or pytorch distribution.
